roka.py

```diff
@@ -2,13 +2,15 @@
 
 import argparse
 import os
-from flask import Flask, request, Response, render_template, send_file
+import shutil
+import json
+from flask import Flask, request, Response, render_template, send_file, templating
+from flask.globals import _app_ctx_stack
 from lib.books import Books
 from lib.util import check_auth, escape, generate_rss, read_cache
 
 abs_path = os.path.dirname(os.path.abspath(__file__))
 app = Flask(__name__)
-app.config.from_pyfile(os.path.join(abs_path, 'app.cfg'))
```

Comment:
If we only populate `app.config` in `__main__` we wouldn't be able to run via uwsgi (key error).

Reply:
Thanks for catching this. I wasn't testing much with uwsgi. I confirmed this is fixed now.